addons/mail/static/src/components/mobile_messaging_navbar/mobile_messaging_navbar.js

```diff
@@ -15,41 +22,20 @@ export class MobileMessagingNavbar extends Component {
      * @param {MouseEvent} ev
      */
     _onClick(ev) {
-        if (this.props.onSelectMobileMessagingNavbarTab) {
-            this.props.onSelectMobileMessagingNavbarTab({
-                tabId: ev.currentTarget.dataset.tabId,
-            });
-        }
+        this.mobileMessagingNavbarView.onClick(ev.currentTarget.dataset.tabId);
     }
 
 }
 
 Object.assign(MobileMessagingNavbar, {
-    defaultProps: {
-        tabs: [],
-    },
     props: {
-        activeTabId: String,
         onSelectMobileMessagingNavbarTab: {
```

Comment:
no longer used?

Reply:
No, it's now a field on `MobileMessagingNavbarView` model.

Reply:
I mean the prop should be removed then.

Reply:
Comment unresolved, prop onSelectMobileMessagingNavbarTab looks no longer used and should be removed.

Reply:
I thought the comment was about `activeTabId`, hence the confusion in my reply.
Indeed, it's no longer used so I just removed props `onSelectMobileMessagingNavbarTab`.